### ISSUE TITLE
Incorrect Caps Lock Indicator Icon

### ISSUE THREAD

**Issue by prat-man** [linuxmint/cinnamon#9971]
```
 * Cinnamon 4.8.6
 * Linux Mint 20.1
 * NVIDIA Graphics; Proprietary driver 460.39
 * 64 bit
 ```

**Issue**
Caps Lock Indicator uses incorrect icon when ON.

**Steps to reproduce**
System Settings > Accessibility > Keyboard > Use visual indicator on Caps and Num Lock: *TURN ON*
Press the Caps Lock key to enable Caps Lock

**Actual behaviour**
Shows upwards arrow icon

**Expected behaviour**
Show complementary icon to Caps Lock disabled icon (locked icon)

**Comment by mtwebster** [comment]
Hi, what icon theme are you using? Can you try both Mint-Y and Mint-X?

**Comment by prat-man** [comment]
I have tried with both Mint-Y and Mint-X.
Additionally, I have tried with the Gnome and Adwaita icon themes as well.
The issue persists across all icon themes.

**Comment by icarter09** [comment]
If I set the Icon Theme to be either Adwaita or Gnome, I see the upwards arrow icon for when Caps Lock is enabled, but see the unlocked icon for it when Caps Lock is disabled. 

But I do not see it when I set to what I normally use for the Icon theme which is Mint-X-Pink. I also do not see it for Mint-X or Mint -Y icon themes either.

**Comment by prat-man** [comment]
That is strange. I see the upwards arrow for all icon themes.
I have tried with Mint-X-Pink, but I always see the upwards arrow icon.

I am using Linux Mint 20.1 Cinnamon, fresh install (not upgraded), if that makes any difference.

**Comment by NikoKrause** [comment]
It has been fixed recently in Mint-X and Mint-Y. It will be available in Mint 20.2. I guess @icarter09 is running the Bleeding Edge Mint PPA

**Comment by icarter09** [comment]
Thanks for the update that it's been addressed @NikoKrause. 

@prat-man I'm closing this ticket since it's been fixed in the next release. Please reopen if you continue to experience it after the next release. Thanks.
